src/interfaces/userInterface.ts

```diff
@@ -0,0 +1,13 @@
+export interface userProps {
```

Comment:
Also, for all types and interfaces, the pattern is for it to be first letter capitalized. Please, change it to UserProps. Also, what is this interface for? If it is a mapping of the User entity from the BE, maybe instead of Props, it could be named properly, for example, UserEntity, or just User.

Reply:
that's true, i changed it to UserProps